dist/_include-media.scss

```diff
@@ -139,6 +222,32 @@ $unit-intervals: (
 ///
 @function noop() {}
 
+@function im-intercepts-static-breakpoint($conditions...) {
+  $show: true;
+  $no-media-breakpoint-value: map-get($breakpoints, $im-no-media-breakpoint);
```

Comment:
What happens if there `$im-no-media-breakpoint` references no existing breakpoint? Shall we throw?